### ISSUE TITLE
Automate image generation

### ISSUE THREAD

**Issue by eloydegen** [schorschii/QuickSync4Linux#5]
[The GIMP steps in the README](QuickSync4Linux?tab=readme-ov-file#picture-format) could be automated by using the ImageMagick wrapper for Python, Wand. The manual alternative should still be available to give the user more control. But it could be useful for flashing a series of images.

**Comment by schorschii** [comment]
Yes, that would be a nice addition. Feel free to open a pull request.
